integ-test/src/test/java/org/opensearch/sql/ppl/MatchBoolPrefixIT.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.sql.ppl;
+
+import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_PHRASE;
+import static org.opensearch.sql.util.MatcherUtils.rows;
+import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
+
+import java.io.IOException;
+import org.json.JSONObject;
+import org.junit.Test;
+
+public class MatchBoolPrefixIT extends PPLIntegTestCase {
+
+  @Override
+  public void init() throws IOException {
+    loadIndex(Index.PHRASE);
+  }
+
+  @Test
+  public void valid_query_match_test() throws IOException {
+    JSONObject result =
+        executeQuery(
+            String.format(
+                "source=%s | where match_bool_prefix(phrase, 'quick') | fields phrase",
```

Comment:
can we change this test to ```match_bool_prefix(phrase, 'qui')``` and it'll still work?  Isn't the last term supposed to be a prefix rather than the full word?

Reply:
See [b6252762cb8b31e990518bf62b6bacc0431ee10c](https://github.com/Bit-Quill/opensearch-project-sql/pull/67/commits/b6252762cb8b31e990518bf62b6bacc0431ee10c) for this fix.